### ISSUE TITLE
Pain at the medial aspect of the knee

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10744]
**Preferred term label:**
Pain at the medial aspect of the knee

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Medial collateral ligament knee injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001154
- label: Pain at the medial aspect of the knee
- parents: [Knee pain (HP:0030839)](https://hpo.jax.org/app/browse/term/HP:0030839)
- pmid: 
- def: n unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the inner side of knee, that is, the region of the knee that faces the other knee.
- comment: This feature may be observed with medial collateral ligament knee injury.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10744
- synonyms:
